### DISCUSSION TITLE
2023/12/16/obsidian-flomo-interact-1/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#8] [General]
# 2023/12/16/obsidian-flomo-interact-1/

对obsidian笔记软件的继续探索。 （备注：有读者需要！（手动滑稽））

https://wz.anoms.top/2023/12/16/obsidian-flomo-interact-1/

**Comment by Hazelandinstant** [comment]
flomo和《潮汐》（一款软件）曾经做过联动，也是非常好的交互方式，只是可惜我没钱
